provisioning/roles/nginx/templates/drupal7-site.j2

```diff
@@ -0,0 +1,61 @@
+{% extends "default-site.j2" %}
+
+{% block index %}
+    index index.php;
+{% endblock %}
+
+{% block extra %}
+    {{ super() }}
+
+    # Seems that the "index" block is not injected from the parent template, add it again here
+    index index.php;
+
+    location = /favicon.ico {
+      log_not_found off;
+      access_log off;
+    }
+
+    location = /robots.txt {
+      allow all;
+      log_not_found off;
+      access_log off;
+    }
```

Comment:
Those two rules seems project specific to me. Also I would say this is something you put in a production configuration, not a development one. What do you think ?